LocalMultiplayerAgent/DeploymentTool/DeploymentScript.cs

```diff
@@ -0,0 +1,270 @@
+﻿using Microsoft.Azure.Gaming.LocalMultiplayerAgent.Config;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using PlayFab;
+using PlayFab.MultiplayerModels;
+using System.IO;
+using Microsoft.Azure.Gaming.LocalMultiplayerAgent.DeploymentTool;
+using Microsoft.Azure.Storage.Blob;
+
+namespace Microsoft.Azure.Gaming.LocalMultiplayerAgent.MPSDeploymentTool
+{
+    public class DeploymentScript
+    {
+        private readonly MultiplayerSettings settings;
+        private readonly DeploymentSettings deploymentSettings;
+        public DeploymentScript(MultiplayerSettings multiplayerSettings)
+        {
+            settings = multiplayerSettings ?? throw new ArgumentNullException(nameof(multiplayerSettings));
+            deploymentSettings = JsonConvert.DeserializeObject<DeploymentSettings>(File.ReadAllText("DeploymentTool/deployment.json"));
+        }
+
+        public async Task RunScriptAsync()
+        { 
+            PlayFabSettings.staticSettings.TitleId = settings.TitleId;
+
+            string secret = Environment.GetEnvironmentVariable("PF_SECRET");
+            if (string.IsNullOrEmpty(secret))
+            {
+                Console.WriteLine("Enter developer secret key");
```

Comment:
We should notify the developer about what is happening here, like "Deploying to PlayFab Multiplayer Servers.... Enter developer secret key to continue:" or something